cli/src/main.rs

```diff
@@ -98,12 +97,12 @@ fn main() {
         remove_processing_instructions: args.remove_processing_instructions,
     });
 
-    if args.inputs.len() < 1 {
+    if args.inputs.is_empty() {
```

Comment:
This should be `len() <= 1`, as otherwise the single-file mode won't work. It looks like my code was wrong too.

Reply:
got it, I will fix this